pkg/storaged/mdraid-details.jsx

```diff
@@ -268,7 +267,7 @@ export class MDRaidDetails extends React.Component {
                 // members.
 
                 function wipe_members() {
-                    return cockpit.all(client.mdraids_members[mdraid.path].map(function (member) {
+                    return Promise.all(client.mdraids_members[mdraid.path].map(function (member) {
                         return member.Format('empty', { });
```

Comment:
Also, this would be an excellent place to use an arrow function. Remember that we now handle *all* of our code consistently as ES6 with Babel, so there is no restriction any more which files can use ES6 syntax.

Reply:
Done. I tried not to change the code too much, even if tempting, so that this change stays isolated.